microservices/provision/src/Services/Provision.php

```diff
@@ -33,7 +33,7 @@ public function execute(
                 ->execute($terminalModel);
         }
 
-        if (! $isHttps) {
+        if (! $isHttps && $_SERVER['APP_ENV'] !== 'test') {
```

Comment:
revert this

Reply:
If I revert this change, the specific provision test will fail, because the logic searches if the request comes from the HTTPS protocol and with a port different from 443.